docker-compose.yml

```diff
@@ -120,9 +117,113 @@ x-hierarchy:
   # helper services
   - impala
   - postgres
+  - python-wheel-manylinux-2010
+  - python-wheel-manylinux-2014
+  - python-wheel-manylinux-test
+  - python-wheel-windows-vs2017
+  - python-wheel-windows-test
+
+volumes:
+  conda-ccache:
+    name: ${ARCH}-conda-ccache
+  debian-ccache:
+    name: ${ARCH}-debian-${DEBIAN}-ccache
+  ubuntu-ccache:
+    name: ${ARCH}-ubuntu-${UBUNTU}-ccache
+  fedora-ccache:
+    name: ${ARCH}-fedora-${FEDORA}-ccache
+  debian-rust:
+    name: ${ARCH}-debian-${DEBIAN}-rust
+  maven-cache:
+    name: maven-cache
+  wheel-manylinux1-ccache:
```

Comment:
How about prepend `python-` because `python-` exists in related names?

Reply:
Updated.